packages/core/upload/admin/src/components/Breadcrumbs/tests/index.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render as renderTL, screen, fireEvent, waitFor } from '@testing-library/react';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { Breadcrumbs } from '../index';
+import en from '../../../translations/en.json';
+
+jest.mock('../../../hooks/useFolderStructure');
+
+jest.mock('../../../utils', () => ({
+  ...jest.requireActual('../../../utils'),
+  getTrad: x => x,
+}));
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  useQueryParams: jest.fn().mockReturnValue([{ query: { folder: 22 } }]),
+}));
+
+jest.mock('react-intl', () => ({
+  useIntl: () => ({ formatMessage: jest.fn(({ id }) => en[id]) }),
+}));
```

Comment:
I recently found out, that we don't need this mocking and better rely on `defaultMessage`. Could you remove it?